packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -440,7 +440,7 @@ const themeStyleTag = computed(
   grid-area: header;
   position: sticky;
   top: var(--scalar-custom-header-height, 0px);
-  z-index: 10;
+  z-index: 1000;
```

Comment:
is this in line with our zIndex strategy @hwkr @cameronrohani ?

Reply:
what do you mean? didn't find much about that so as a quick fix since it's in prod i increased it